nutrition_labels/prodigy_training_data.py

```diff
@@ -43,7 +43,8 @@
                 training_data.append(annotation)
 
     training_data = pd.DataFrame(training_data)
-    training_data.drop_duplicates(subset=['Internal ID']).reset_index()
+    training_data.drop_duplicates(subset=['Internal ID'], inplace=True)
```

Comment:
this changes the result from before, was there a bug?

Reply:
yep :( the results of 210218 which are all commented out are from this bug